core/database/foxx/tests/user_router.test.js

```diff
@@ -0,0 +1,130 @@
+"use strict"
+// NOTE: completion of tests requires successful run of user_fixture.js script
+
+// Integration test of API
+const { expect } = require("chai");
+const request = require("@arangodb/request");
+const { baseUrl } = module.context;
+const { db } = require("@arangodb");
+
+const usr_base_url = `${baseUrl}/usr`
+
+
+// NOTE: describe block strings are compared against test specification during test call, not file name
+describe("user_router: the Foxx microservice user_router token/set endpoint", () => {
+    const test_param_indexes = [0,1,2,3,4];
+    const test_params = test_param_indexes.map((param_index) => {
+        return {
+            _key: "testUser" + param_index,
+            access: "access_token" + param_index,
+            refresh: "refresh_token" + param_index,
+        }
+    });
+    const test_edge_params = {
+        type: 4,  // transfer token
```

Comment:
Where are these numbers defined? These should probably be an enum instead of a magic number, unless I'm missing something. They could be defined in the foxx/api/support.js file with the other enums.

Reply:
Nevermind, I now understand see earlier comment.